src/Chapter1.hs

```diff
@@ -554,7 +557,10 @@ value after "=" where the condition is true.
 Casual reminder about adding top-level type signatures for all functions :)
 -}
 
-mid x y z = error "mid: not implemented!"
+mid ::  Int -> Int -> Int -> Int
+mid x y z = 
+  (x + y + z) - (max x (max y z)) - (min x (min y z))
```

Comment:
Interesting solution! Though, not sure if it works for all numbers because of integer overflow :thinking: